cobc/typeck.c

```diff
@@ -3879,23 +3879,26 @@ validate_alphabet (cb_tree alphabet)
 		int			lastval = 0, tableval = 0;
 		int			pos = 0;
 		int			i;
-		int			values[256];
-		int			charvals[256];
-		int			dupvals[256];
+		const int maxchar = ap->alphabet_target == CB_ALPHABET_ALPHANUMERIC
+			? 256: 65536;
+		int			values[maxchar];
+		int			charvals[maxchar];
+		int			dupvals[maxchar];
 
-		for (n = 0; n < 256; n++) {
+		for (n = 0; n < maxchar; n++) {
 			values[n] = -1;
 			charvals[n] = -1;
 			dupvals[n] = -1;
 			ap->values[n] = -1;
 			ap->alphachr[n] = -1;
 		}
```

Comment:
As mentioned in the call this needs to be fixed to be portable (the variable length arrays [VLAs] are not C90 standard code).
I'll try implementing the support for national alphabets "upstream", fixing this while improving both the amount of storage and performance. Therefore I mark all of the related review-comments as "resolved" (= other than understanding them in principle there is nothing to do for you there), allowing you to concentrate on the other "non-resolved" ones and trying to add the command line options.

I'll drop a note as soon this is is available (expected until tomorrow), allowing you to revert the parts locally and rebase (we can do that together) - after pushing that there should also be more CI running again.